pkg/tnf/interactive/spawner.go

```diff
@@ -272,6 +281,23 @@ func NewGoExpectSpawner() *GoExpectSpawner {
 	return &GoExpectSpawner{}
 }
 
+// LogStdErr logs stderr output to logger (warning)
+func LogStdErr(cmdLine string, stderrpipe io.Reader) {
+	buf := bufio.NewReader(stderrpipe)
+
+	for {
+		line, _, err := buf.ReadLine()
+		log.Warn("STDERR for " + cmdLine + " : " + string(line))
+
+		if err != nil {
+			// Some Error has happened, goroutine about to exit
+			log.Warn(err)
```

Comment:
ditto

Reply:
I think this is actually an error so I would go with log.Error here.

Reply:
This is how we exit this go routine for instance when the discovery recompute the container under test. There is an error reading the pipe. It is not really an error execpt maybe in certain cases

Reply:
You're right.